src/test/java/org/junitpioneer/jupiter/issue/IssueExtensionIntegrationTests.java

```diff
@@ -62,7 +62,7 @@ private static void assertStatus(IssueTestCase testCase) {
 			assertThat(testCase.result()).isEqualTo(Status.FAILED);
 	}
 
-	static class IssueIntegrationCases {
+	static class IssueIntegrationTestCase {
```

Comment:
Should this be "...TestCase<ins>s</ins>"? (with "s" at the end)

Reply:
Yes, good catch. 👍🏾